src/coreclr/jit/lowerxarch.cpp

```diff
@@ -2124,6 +2191,55 @@ GenTree* Lowering::LowerHWIntrinsicCreate(GenTreeHWIntrinsic* node)
             return LowerNode(node);
         }
 
+        // We have the following (where simd is simd16, simd32 or simd64):
+        //          /--*  op1  T
+        //   node = *  HWINTRINSIC   simd   T Create
+
+        if (intrinsicId == NI_Vector512_Create)
+        {
+            assert(comp->compIsaSupportedDebugOnly(InstructionSet_AVX512F));
```

Comment:
Same here as previous places. We should likely have a `IsBaselineVector512IsaSupportedDebugOnly()` check that makes this simple and that can account for `BW` + `DQ`.

Reply:
Done